yarpcconfig/chooser_test.go

```diff
@@ -363,6 +364,24 @@ func TestChooserConfigurator(t *testing.T) {
 				_ = list
 			},
 		},
+		{
+			desc: "use fewest-pending-requests chooser",
```

Comment:
The test right above this one which uses the `peer/x/peerheap` and has a very similar name. Is `pendingheap` intended to deprecate the experimental peerheap? Reading the docs, I can't really tell the difference either:
[pendingheap](https://godoc.org/go.uber.org/yarpc/peer/pendingheap)
[peerheap](https://godoc.org/go.uber.org/yarpc/peer/x/peerheap)

Reply:
`x/` packages are experimental and I'm not entirely sure of the history behind the `x/peerheap` but I wouldn't be surprised if that turned into the `pendingheap`.

@kriskowal can add some context / maybe we remove it in a follow up PR?